miden-lib/asm/sat/kernel.masm

```diff
@@ -5,6 +5,16 @@ use.miden::sat::internal::layout
 use.miden::sat::internal::note
 use.miden::sat::internal::tx
 
+# EVENTS
+# =================================================================================================
+
+# Event emitted to signal that an asset is being added to the account vault.
+const.ADD_ASSET_TO_ACCOUNT_VAULT_EVENT=0
+
+# Event emitted to signal that an asset is being removed from the account vault.
+const.REMOVE_ASSET_FROM_ACCOUNT_VAULT_EVENT=1
```

Comment:
Let's set event ID values using the following scheme:

- The most significant 16 bits should be set to $2$ to signify that the even is coming from `midenlib`.
- The least significant 16 bits should be set to the event ID - i.e., $0$ and $1$.

If I did my math correctly, this would set these event values to $131072$ and $131073$.